### PR TITLE
Feature/display player

### PR DESCRIPTION
**Pull request by AbdullahNanaDVT**
Allows the user to view all the players in the squad for each team.

Simulator Screen Shot - iPhone 13 Pro - 2021-11-16 at 11 52 03

### CODE REVIEW

**Review by VincentMoyo — APPROVED** [review]

File: Kick-Starter/Kick-Starter/Display Players/DisplayPlayersViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+//  Kick-Starter
+//
+//  Created by Abdullah Nana on 2021/11/15.
+//
+
+import Foundation
+import KickStarterFramework
+
+final class DisplayPlayersViewModel {
+    private(set) var playerResponse: PlayerResponseModel?
```
Comment (bgouws) [inline Kick-Starter/Kick-Starter/Display Players/DisplayPlayersViewModel.swift:12]: Do these need to be `private(set)` ?

File: Kick-Starter/Kick-Starter/Teams Information/TeamsInformationViewController.swift [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import UIKit
+import KickStarterFramework
+
+class TeamsInformationViewController: UIViewController {
+    private lazy var teamsInformationViewModel = TeamsInformationViewModel()
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    @IBAction func didTapVenueButton(_ sender: Any) {
```
Comment (bgouws) [inline Kick-Starter/Kick-Starter/Teams Information/TeamsInformationViewController.swift:17]: Private

File: Kick-Starter/Kick-Starter/Teams Information/TeamsInformationViewController.swift [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        performSegue(withIdentifier: "displayVenueSegue", sender: self )
+    }
+    
+    @IBAction func didTapPlayersButton(_ sender: Any) {
```
Comment (bgouws) [inline Kick-Starter/Kick-Starter/Teams Information/TeamsInformationViewController.swift:21]: Private

File: Kick-Starter/Kick-Starter/Teams Information/TeamsInformationViewModel.swift [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+//  TeamsInformationViewModel.swift
+//  Kick-Starter
+//
+//  Created by Abdullah Nana on 2021/11/15.
+//
+
+import Foundation
+import KickStarterFramework
+
+final class TeamsInformationViewModel {
```
Comment (bgouws) [inline Kick-Starter/Kick-Starter/Teams Information/TeamsInformationViewModel.swift:11]: You can add some line breaks in this class to make things a little more readable 😄

**Review by bgouws — CHANGES_REQUESTED** [review]
Just some clean up 😄

**Review by bgouws — APPROVED** [review]

**Comment by bgouws** [comment]
Just a note but you could add a title to this screen as well 😄

**Pull request merged**